source/javascripts/meetups.js

```diff
@@ -44,37 +64,35 @@
   locations.forEach( generateMarkerData );
 
   handler.buildMap(mapOptions, function() {
```

Comment:
@wycats @sandraor Thoughts on how to resolve? Presumably we need to pick one and adapt the other PR to accommodate.